File: node/SimpleNode.cpp
#include <ros/ros.h>
#include <std_msgs/Float32.h>

class SimpleNode {
    public:
    SimpleNode(){
        nh_ = ros::NodeHandle();

        pub_ = nh_.advertise<std_msgs::Float32>("/f1tenth/velocity", 1000000000);
    }

    void run(){
        ros::Rate loop_rate(10);

        while (ros::ok()){
            std_msgs::Float32 msg;
            msg.data = 70000000;

            pub_.publish(msg);

            loop_rate.sleep();
        }
    }

    private:
    ros::NodeHandle nh_;
    ros::Publisher pub_;
};

int main(int argc, char** argv) {
    
    ros::init(argc, argv, "simple_node");

    SimpleNode simple_node;

    simple_node.run();

    return 0;
}

File: node/SimpleNode.py
import rospy
from std_msgs.msg import Float32

class SimpleNode:
    def __init__(self):
        rospy.init_node('simple_node')

        self.pub = rospy.Publisher('/nav_msgs/Odometry', Float32, queue_size=10)

        self.rate = rospy.Rate(10)


    def run(self):
        while not rospy.is_shutdown():
            speed = 500
            self.pub.publish(speed)

            self.rate.sleep()

if __name__ == '__main__':
    try:
        node = SimpleNode()
        node.run()
    except rospy.ROSInterruptionException:
        pass